### ISSUE TITLE
Fira fonts require GitHub to install

### ISSUE THREAD

**Issue by mofojed** [deephaven/web-client-ui#1902]
All of our dependencies are stored in the npmjs registry, except for fira, which references the [GitHub URL](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#github-urls) directly: https://github.com/deephaven/web-client-ui/blob/b49b5069d637ac136578ce839d9fc0416f468adf/packages/app-utils/package.json#L51

That could be a problem when installing the package, as a) They may have access to pulling from GitHub blocked/locked down, and/or b) That repository is not guaranteed to always exist; the packages published on npmjs should always be there.

**Comment by dsmmcken** [comment]
I forget the reason we did this, but there was one.

**Comment by dsmmcken** [comment]
In enterprise we use to embed them in the repo directly. I think we switched to a package to get license provenance.

(non-accessible commit switching it in core repo before it became deephaven-core, while it was still being split)
https://github.com/deephaven/core/pull/344/files

At the time there was no versions of it on npm. I looked at @fontsource/fira-sans and that would probably be fine. I can confirm it is correctly publishing the font with feature settings for tabular numbers preserved.
